packages/design-system-twrnc-preset/src/Theme/Theme.utilities.ts

```diff
@@ -1,20 +1,18 @@
-import baseConfig from '../../tailwind.config';
```

Comment:
Removing the empty tailwind config import as it doesn't serve any purpose as far as I'm aware